### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by 1forh**
See commits

### CODE REVIEW

File: assets/scripts/base/detect.js [hunk 1]
```diff
@@ -51,9 +52,13 @@ export class FireDetect {
...
     document.documentElement.style.setProperty('--vh', `${vh}px`);
 
-    $(window).on('resize', () => {
-      vh = window.innerHeight * 0.01;
-      document.documentElement.style.setProperty('--vh', `${vh}px`);
-    });
+    $(window).on(
+      'resize',
+      throttle(() => {
+        console.log('hit');
```
Comment (wuebkers) [inline assets/scripts/base/detect.js]: console.log

File: assets/styles/admin/_local-tasks.scss [hunk 2]
```diff
@@ -2,7 +2,7 @@
   padding: 12px 0;
   display: flex;
   justify-content: center;
-  background-color: $color-gray;
+  background-color: $gray-900;
```
Comment (BrentWMiller) [inline assets/styles/admin/_local-tasks.scss]: Can also call this with `gray('900')`

Comment (BrentWMiller) [reply assets/styles/admin/_local-tasks.scss]: Same in other spots. I'm fine with this though.

Comment (1forh) [reply assets/styles/admin/_local-tasks.scss]: We should probably stick with one or the other. I'm fine with either though. The functions do seem nicer but there's not a huge difference than I can think of.

Comment (BrentWMiller) [reply assets/styles/admin/_local-tasks.scss]: Yeah I honestly went back and forth on this. The variables auto-complete while the function calls don't. The maps look cleaner, but the variables are easier to use.

Comment (1forh) [reply assets/styles/admin/_local-tasks.scss]: I think we should stick with the easier to use option. Auto-complete is really nice when working with this stuff.

**Pull request merged**